### ISSUE TITLE
ListView item text doesn't update

### ISSUE THREAD

**Issue by TheBolliwood** [halfnelson/svelte-native#44]
## Expected Behavior
ListView displays the complete item array correctly.

## Actual Behavior
ListView only displays the text of the initial items.

## Steps to Reproduce the Problem

  1. [Do the Tutorial till the "Add Tasks" functionality](https://svelte-native.technology/tutorial#basic-functionality-add-tasks)
  2. Type something into the textField (I typed foo, bar and baz)
  3. The ListView only displays foo, but the console shows also bar and baz

![Screenshot_1564944686](https://user-images.githubusercontent.com/11932018/62427864-d3eac080-b6f9-11e9-8a3b-bba4b4b66313.png)

![image](https://user-images.githubusercontent.com/11932018/62427693-14950a80-b6f7-11e9-9f2f-c0033e732cab.png)

## Specifications

  - tns Version: 6.0.2

Device 1:
  - Android Version: 9.0 (Pie) - API 28
  - Emulator Version: 29.0.9-5537588
  - Kernel Version: Linux 5.0.0-21-generic x86_64
  - Platform: Ubuntu 19.04

Device 2:
  - Android Version: 9.0 (Pie) - API 28
  - NativeScript Playground Version: 1.8.0
  - NativeScript Preview Version: 1.23.0 

Device 3:
  - Android Version: 9.0 (Pie) - API 28
  - Emulator Version: 29.0.11-5598178
  - Kernel Version: Linux 4.15.0-48-generic x86_64
  - Platform: Ubuntu 18.04 LTS

**Comment by halfnelson** [comment]
Thanks for this. Looks like svelte's dirty check generated code changed from 3.6.1 to 3.6.2 which causes the item text not to update. Not sure if that was intended and I need to fix svelte-native or whether it was unintended and I need to ticket against svelte. I'll investigate.

**Comment by halfnelson** [comment]
for reference the code went from 
```js
	p(changed, ctx) {
			if (default_slot && default_slot.p && (changed.$$scope || changed.props)) {
				default_slot.p(get_slot_changes(default_slot_1, ctx, changed, get_default_slot_changes), get_slot_context(default_slot_1, ctx, get_default_slot_context));
			}
		},
```
to 
```js
	if (default_slot && default_slot.p && changed.$$scope) {
				default_slot.p(get_slot_changes(default_slot_1, ctx, changed, get_default_slot_changes), get_slot_context(default_slot_1, ctx, get_default_slot_context));
			}
```

notice it no longer checks for changed.props which is what we are changing.

**Comment by halfnelson** [comment]
I have reproduced this bug in svelte 3.7.0, so in the short term to work around this, I will build a new svelte-native with 3.6.1. (the last working version)

**Comment by halfnelson** [comment]
Svelte bug for tracking https://github.com/sveltejs/svelte/issues/3354

**Comment by halfnelson** [comment]
I have fixed this with svelte-native 0.4.1
ensure all references in package.json are to svelte 3.6.1
